content/20.treatments.md

```diff
@@ -46,10 +46,79 @@ Antivirals have been used to treat XYZ diseases through [what mechanisms or appr
 
 Add a subcategory (using ####) for each category of antiviral treatment
 
-#### Antiviral Approach 1
+#### Viral Protein Targeting Drugs
 
 Brief background on the therapeutic.
 
+##### Nucleoside Analogues
+
+Why it may be useful
+
+###### Current Evidence
+
+A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
+
+###### Summary
+
+Summarize the state of the antiviral approach.
+
+##### Protease Inhibitors
+
+Why it may be useful
+
+###### Current Evidence
+
+A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
+
+###### Summary
+
+Summarize the state of the antiviral approach.
+
+##### Viral Envelope
+
+Why it may be useful
+
+###### Current Evidence
+
+A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
+
+###### Summary
+
+Summarize the state of the antiviral approach.
+
+#### Host Protein Targeted Drugs
+
+Brief background on the therapeutic.
+
+##### Viral Entry Receptors
+
+Why it may be useful
+
+###### Current Evidence
+
+A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
+
+###### Summary
+
+Summarize the state of the antiviral approach.
+
+#### Broad-Spectrum Pharmaceuticals
+
+##### Nutraceuticals
+
+##### Hydroxychloroquine
+
+### Biological Drugs for COVID-19
+
+#### Antibodies
+
+Monoclonal antibodies (mAbs) have revolutionized the way we treat human diseases and have become the best-selling drugs in the pharmaceutical market.
```

Comment:
Is there a recent review paper on this that you can cite? Just curious since it's a pretty strong statement. The sentences that follow provide a couple cases but there's little about how those drugs impact outcomes, etc (granted - that _shouldn't_ go into our paper but citing a review here would help folks find that type of info).

Reply:
Hi! I re-worded that intro a little bit in my word document already because I think that was a little bit of an overstatement. This is a review from this year : doi: 10.1186/s12929-019-0592-z. I think I should reword it do make it sound less dramatic maybe.

Reply:
Can you use the three dots at the top of this file -> edit file and change it here as well? Thanks!

Reply:
Done!